schemas.js

```diff
@@ -34,7 +34,7 @@ function mongoSchemaAPIResponseFields(schema) {
   fieldNames = Object.keys(schema).filter(key => key !== '_id');
   response = {};
   fieldNames.forEach(fieldName => {
```

Comment:
prefer reduce over forEach to avoid side effects.

```response = fieldNames.reduce( obj, fieldName => {
obj[fieldName] = mongoFieldTypeToSchemaAPIType(schema[fieldName]);
}, {})```